objects.js

```diff
@@ -0,0 +1,99 @@
+"use strict";
+
+/*
+1. Picture gallery
+*/
+function getLeftValueForImage(images, widths, imageIndex) {
+    let result = 0;
+    for (let i =0 ; i < imageIndex; i++) {
+        result-=widths[i]
+    }
+    return result;
+}
+
+const images = ['1.jpg', '2.jpg', '3.jpg'];
+const widths = [200, 300, 400];
+
+let leftValue = getLeftValueForImage(images, widths, 0);
+console.log(`Left value should be 0: ${leftValue}`);
+leftValue = getLeftValueForImage(images, widths, 1);
+console.log(`Left value should be -200: ${leftValue}`);
+leftValue = getLeftValueForImage(images, widths, 2);
+console.log(`Left value should be -500: ${leftValue}`);
+leftValue = getLeftValueForImage(images, widths, 3);
+console.log(`Left value should be -900: ${leftValue}`);
+
+function compareCars(car1, car2) {
+    if (car1.speed > car2.speed) {
+        return car1;
+    }
+    return car2;
+}
+
+/*
+2. Fastest car
+*/
+function getFastestCar(cars) {
+    return cars.reduce(compareCars);
```

Comment:
Instead of 2 long functions, you can do use the short `sort` method:
```js
const winner = cars.sort((car1, car2) => car2.speed - car1.speed)[0];
```